src/platform/native/Core.zig

```diff
@@ -537,6 +537,9 @@ pub fn deinit(self: *Core) void {
 pub fn appUpdateThread(self: *Core, app: anytype) void {
     self.frame.start() catch unreachable;
     while (true) {
+        const pool = util.AutoReleasePool.init() catch unreachable;
+        defer util.AutoReleasePool.release(pool);
```

Comment:
What's the reason for this @pdoane ? Dawn used to require this but AFAIK no longer does, so I actually removed it. It's not great to have this from a C ABI perspective since it's so specific to objc

Reply:
Yeah, I can look at alternatives. Typically I’ve seen this handled outside of the gfx stack as there maybe more objc interaction but we can probably isolate it.

Reply:
autoreleasepool removed, dusk changes incoming soon